conf/provisioning/dashboards/nuodb/communication.json

```diff
@@ -5,10 +5,12 @@
         "$$hashKey": "object:21",
         "builtIn": 1,
         "datasource": "-- Grafana --",
-        "enable": true,
```

Comment:
having default annotations on leaves annotation only where you place it.   I rather have control.  db annotations is the control is tag by database name.  Then if you filter by database it will only show those annotations.     It does not allow annotations from different databases.

Reply:
I understand that you can't use annotation and have database set to multiple (or all) databases.